docs/visual-basic/language-reference/statements/on-error-statement.md

```diff
@@ -24,94 +24,92 @@ helpviewer_keywords:
 ms.assetid: ff947930-fb84-40cf-bd66-1ea219561d5c
 ---
 # On Error Statement (Visual Basic)
-Enables an error-handling routine and specifies the location of the routine within a procedure; can also be used to disable an error-handling routine.  
-  
- Without error handling, any run-time error that occurs is fatal: an error message is displayed, and execution stops.  
-  
- Whenever possible, we suggest you use structured exception handling in your code, rather than using unstructured exception handling and the `On Error` statement. For more information, see [Try...Catch...Finally Statement](../../../visual-basic/language-reference/statements/try-catch-finally-statement.md).  
-  
+Enables an error-handling routine and specifies the location of the routine within a procedure; can also be used to disable an error-handling routine.
+
+ Without error handling, any run-time error that occurs is fatal: an error message is displayed, and execution stops.
+
 > [!NOTE]
->  The `Error` keyword is also used in the [Error Statement](../../../visual-basic/language-reference/statements/error-statement.md), which is supported for backward compatibility.  
-  
-## Syntax  
-  
-```  
-On Error { GoTo [ line | 0 | -1 ] | Resume Next }  
-```  
-  
-## Parts  
-  
-|Term|Definition|  
-|---|---|  
-|`GoTo` `line`|Enables the error-handling routine that starts at the line specified in the required `line` argument. The `line` argument is any line label or line number. If a run-time error occurs, control branches to the specified line, making the error handler active. The specified line must be in the same procedure as the `On Error` statement, or a compile-time error will occur.|  
-|`GoTo` 0|Disables enabled error handler in the current procedure and resets it to `Nothing`.|  
-|`GoTo` -1|Disables enabled exception in the current procedure and resets it to `Nothing`.|  
-|`Resume Next`|Specifies that when a run-time error occurs, control goes to the statement immediately following the statement where the error occurred, and execution continues from that point. Use this form rather than `On Error GoTo` when accessing objects.|  
-  
-## Remarks  
-  
+>  The `Error` keyword is also used in the [Error Statement](../../../visual-basic/language-reference/statements/error-statement.md), which is supported for backward compatibility.
+
+## Syntax
+
+```vb
+On Error { GoTo [ line | 0 | -1 ] | Resume Next }
+```
+
+## Parts
+
+|Term|Definition|
+|---|---|
+|`GoTo line`|Enables the error-handling routine that starts at the line specified in the required `line` argument. The `line` argument is any line label or line number. If a run-time error occurs, control branches to the specified line, making the error handler active. The specified line must be in the same procedure as the `On Error` statement, or a compile-time error will occur.|
```

Comment:
@rpetrusha How would we best indicate here that line is a placeholder for whatever line you want.

Reply:
The best way is probably \`GoTo` \*line\*. The fenced lines in the description should also be changed to \*line\*.